### DISCUSSION TITLE
Ability to use groupings within the permissions screen

### DISCUSSION THREAD

**Discussion by TLMondo** [directus/directus#19129] [Feature Requests]
### Summary

When assigning permissions, we have 50+ Collections, and will have at least a few hundred more. Like we can group Collections in the `Content` and `Data Model` screens, we would like the same in Permissions. This way we can "hide" hundreds out of view.

<img width="825" alt="image" src="https://github.com/directus/directus/assets/7502754/fee7351b-a40b-40ec-8ada-53ff7bcfcc05">

### Motivation

Painful to scroll through hundreds of Collections, when you only need to work within a Group most of the time.

### Detailed Design

Simply replicate UI pattern from other sections.

### Must Have:
- Collections should follow the same groupings created in the `Content` and/or `Data Model` sections

### Should Have:
- Collapse all groups by default

### Could Have:
- The ability to set custom sorts for Groups, per user

### Won't Have:
- 

### Drawbacks

Shouldn't really have any drawbacks, as this is the convention in other sections.

### Alternatives

Search/Filter text field at the top of the list
- Support `tags` for Collections, ie. "user_related"

### Adoption Strategy

Should be simple enough to maybe put a 'tour tip' the first time they return to the section

**Comment by u12206050** [comment]
Would an alternative to this be to have a default Authenticated Role just like the Public Role, on which you can set permissions that would apply to all authenticated users, kind of like the default permissions of a collection.
